learn/app-development/deployment/build-with-docker.md

```diff
@@ -12,9 +12,9 @@ For creating a Docker container, you create a Docker image by building a Dockerf
 
 |Description|Version|
 |---|---|
-|Java |1.8|
+|Java |11|
 |Node|12.22|
-|Maven| 3.8|
+|Maven| 3.8.6|
```

Comment:
https://pr-649.d2ulx3keu6fdjg.amplifyapp.com/learn/app-development/deployment/building-with-maven#system-prerequisites

Add system requirements in one place and manage better..
https://pr-649.d2ulx3keu6fdjg.amplifyapp.com/learn/wavemaker-release-notes/v11-1-2#build-environment-for-multi-version-studio